pooch/downloaders.py

```diff
@@ -100,9 +107,22 @@ def __call__(self, url, output_file, pooch):
         try:
             response = requests.get(url, **kwargs)
             response.raise_for_status()
-            for chunk in response.iter_content(chunk_size=1024):
-                if chunk:
-                    output_file.write(chunk)
+            if self.progressbar:
+                total_size = int(response.headers.get("content-length", 0))
+                with tqdm.tqdm(
+                    response.iter_content(1024),
+                    total=total_size,
+                    unit="B",
+                    unit_scale=True,
+                ) as bar:
+                    for chunk in response.iter_content(chunk_size=1024):
+                        if chunk:
+                            bar.update(len(chunk))
+                            output_file.write(chunk)
+            else:
+                for chunk in response.iter_content(chunk_size=1024):
```

Comment:
Now that I see it repeated here, we should probably have `chunck_size` as a variable to avoid repetition.